it/mail/chrome/messenger/messenger.properties

```diff
@@ -188,15 +188,15 @@ documentDone=
 pbDocumentLoading=
 
 #LOCALIZATION NOTE #1 is the total number of messages in the selected folder
-totalMsgTitle=(#1 message);(#1 messages)
+totalMsgTitle=(#1 messaggio);(#1 messaggio)
```

Comment:
CORRECTION NEEDED. I guess the first word between brackets should represent the singular form (message), while the second one should represent the plural form (messages). Please replace with the following: 

 +totalMsgTitle=(#1 messaggio);(#1 messaggi)